### PR TITLE
[doc] Try a few ≠ ways of linking to Papers, users, and org profiles

### PR DESCRIPTION
**Pull request by julien-c**
would love to hear others' thoughts.

### CODE REVIEW

**Comment by julien-c** [comment]
for some reason i can't see the model pages in the PR's generated doc

**Comment by sgugger** [comment]
I can't preview the doc for some reason (getting an error on the [distilbert preview page](https://moon-ci-docs.huggingface.co/docs/transformers/pr_22611/en/model_doc/distilbert)). Also there seems to be an issue with your CircleCI permissions, the tests won't run.
Could you try refreshing your permissions as shown [here](https://support.circleci.com/hc/en-us/articles/360048210711-How-to-Refresh-User-Permissions-)?

Otherwise the changes look good to me in preview. Would just love to preview the badge for papers!

**Comment by LysandreJik** [comment]
Weird, the doc build ran successfully and uploaded a zip file, but it does not contain the modeling files. Will see what's up in a bit.

**Comment by LysandreJik** [comment]
It's correctly displayed on the docs now and on the link you shared @sgugger, there was an error with the backend sync.

**Comment by julien-c** [comment]
ok i kinda like it, WDYT?

image

Link: https://moon-ci-docs.huggingface.co/docs/transformers/pr_22611/en/model_doc/t5

**Review by sgugger — APPROVED** [review]
Nice!

File: docs/source/en/model_doc/distilbert.mdx [hunk 1]
```diff
@@ -19,13 +19,16 @@ specific language governing permissions and limitations under the License.
...
+<img alt="Paper page" src="https://img.shields.io/badge/Paper%20page-1910.01108-green">
+</a>
 </div>
 
 ## Overview
 
 The DistilBERT model was proposed in the blog post [Smaller, faster, cheaper, lighter: Introducing DistilBERT, a
 distilled version of BERT](https://medium.com/huggingface/distilbert-8cf3380435b5), and the paper [DistilBERT, a
-distilled version of BERT: smaller, faster, cheaper and lighter](https://arxiv.org/abs/1910.01108). DistilBERT is a
+distilled version of BERT: smaller, faster, cheaper and lighter](https://arxiv.org/papers/1910.01108). DistilBERT is a
```
Comment (stevhliu) [inline docs/source/en/model_doc/distilbert.mdx:31]: Did you mean to link it here?

```suggestion
distilled version of BERT: smaller, faster, cheaper and lighter](https://huggingface.co/papers/1910.01108). DistilBERT is a
```

**Review by stevhliu — APPROVED** [review]
Super cool! ✨

File: docs/source/en/model_doc/distilbert.mdx (hunk 1 above)
Comment (MKhalusova) [reply docs/source/en/model_doc/distilbert.mdx:31]: Agree that this link should either be the HF link, or, alternatively, [a link to the paper pdf](https://arxiv.org/pdf/1907.11692)

**Review by MKhalusova — APPROVED** [review]
I like having the links to HF papers in model docs. I have one question though: should we somehow distinguish links to HF papers from links to arxiv?

**Pull request merged**